src/module_3/module_3.md

```diff
@@ -0,0 +1,891 @@
+```python
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+```
+
+
+```python
+df = pd.read_csv('/Users/carlospujades/Documents/Carletes/Myself/Zrive/Module 2/feature_frame.csv')
```

Comment:
A short explanation of the df (what are the rows, what are the columns...) would be great to refresh our memory